### PR TITLE
Enable build for arm and arm64

### PR DESCRIPTION
**Pull request by bingbing8**
Eanble arm64 and arm build

### CODE REVIEW

File: contrib/win32/openssh/OpenSSHBuildHelper.psm1 [hunk 1]
```diff
@@ -494,27 +487,42 @@ function Start-OpenSSHBuild
...
         $xml.Project.PropertyGroup.AdditionalDependentLibs = 'onecore.lib'
+        $xml.Project.PropertyGroup.MinimalCoreWin = 'true'
         $xml.Save($PathTargets)
     }
-
-    $msbuildCmd = "msbuild.exe"
+    
     $solutionFile = Get-SolutionFile -root $repositoryRoot.FullName
     $cmdMsg = @("${solutionFile}", "/p:Platform=${NativeHostArch}", "/p:Configuration=${Configuration}", "/m", "/noconlog", "/nologo", "/fl", "/flp:LogFile=${script:BuildLogFile}`;Append`;Verbosity=diagnostic")
+    $msbuildCmd = Get-VS2015BuildToolPath
```
Comment (bagajjal) [inline contrib/win32/openssh/OpenSSHBuildHelper.psm1]: create else block and move this to else block, line 524

File: contrib/win32/openssh/config.vcxproj [hunk 2]
```diff
@@ -1,243 +1,459 @@
...
-  </ItemDefinitionGroup>
-  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
-  <ImportGroup Label="ExtensionTargets">
-  </ImportGroup>
-  <Target Name="BeforeClean">
-    <Delete Files="$(OpenSSH-Src-Path)contrib\win32\win32compat\inc\crtheaders.h" />
-  </Target>
-</Project>
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
Comment (bagajjal) [inline contrib/win32/openssh/config.vcxproj:2]: ARM/ARM64 is supported only on visual studio 2017, why can't we move our solution from VS2015 (ToolsVersion="14.0") to VS2017 (ToolsVersion="15.0") completely?

Comment (bingbing8) [reply contrib/win32/openssh/config.vcxproj:2]: 1. for most existing users, they only need x64 and x86, there is no need for them to move to 2017 if not necessary.
2. the version we required for 2017 is not available on chocolatey

File: contrib/win32/openssh/OpenSSHBuildHelper.psm1 (hunk 1 above)
Comment (bingbing8) [reply contrib/win32/openssh/OpenSSHBuildHelper.psm1]: updated.

**Review by bagajjal — APPROVED** [review]

**Pull request merged**